src/script/standard.cpp

```diff
@@ -25,7 +25,7 @@ WitnessV0ScriptHash::WitnessV0ScriptHash(const CScript& in)
     CSHA256().Write(in.data(), in.size()).Finalize(begin());
 }
 
-const char* GetTxnOutputType(txnouttype t)
+std::string GetTxnOutputType(txnouttype t)
```

Comment:
This function has `return nullptr;` statement. It seems wrong :)

Reply:
Yeah should be changed. Though I'm not sure it should be an assert or if it should return an Optional as stated [here](https://github.com/bitcoin/bitcoin/issues/19000#issuecomment-630228064)

Reply:
An assert seems appropriate, as this is unreachable code

Reply:
[Developer Notes - C++ data structures](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#c-data-structures) could be useful :)

Reply:
Changed to assert